xbmc/Application.cpp

```diff
@@ -1978,7 +1979,34 @@ bool CApplication::OnAction(const CAction &action)
     return true;
   }
 
-  if ((action.GetID() == ACTION_INCREASE_RATING || action.GetID() == ACTION_DECREASE_RATING) && m_appPlayer.IsPlayingAudio())
+  if ((action.GetID() == ACTION_SET_RATING) && m_appPlayer.IsPlayingAudio())
+  {
+    const CMusicInfoTag *tag = g_infoManager.GetCurrentSongTag();
+    if (tag)
+    {
+      //Update with any rating changes to playing song already made via GUI e.g. on song info dialog
```

Comment:
no action should directly upate GUIInfoManager. route all changes to Capp

Reply:
Again, I do not see a way to prevent the GUI item from changing.

Reply:
it is only code and can be fixed. search the location that update guiInfo directly and fix it.